test/parallel/test-util-inspect.js

```diff
@@ -570,22 +570,23 @@ assert.doesNotThrow(() => {
 // util.inspect should not display the escaped value of a key.
 {
   const w = {
-    '\\': 1,
-    '\\\\': 2,
-    '\\\\\\': 3,
-    '\\\\\\\\': 4,
+    '\\': 2,
+    '\\\\': 4,
+    '\\\\\\': 6,
+    '\\\\\\\\': 8,
   };
 
   const y = ['a', 'b', 'c'];
-  y['\\\\\\'] = 'd';
+  y['\\\\'] = 'd';
 
   assert.strictEqual(
     util.inspect(w),
-    '{ \'\\\': 1, \'\\\\\': 2, \'\\\\\\\': 3, \'\\\\\\\\\': 4 }'
+    '{ \'\\\\\': 2, \'\\\\\\\\\': 4, \'\\\\\\\\\\\\\': 6, ' +
+    '\'\\\\\\\\\\\\\\\\\': 8 }'
   );
   assert.strictEqual(
```

Comment:
Suggestion, could you add a test with `\r` and `\n`

Reply:
thanks for your suggestion, I will add it later